### PR TITLE
[Table/Collection Fixes] Clearing correctly, no mutating options, no duplicate requests

### PR DESCRIPTION
**Pull request by jvigliotta**
Fixes a couple issues, all in the title.

closes: #4167 
closes: #4202

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? Will users need to change how they are calling the API, or how they've extended core plugins such as Tables or Plots?

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: src/plugins/timeConductor/ConductorInputsFixed.vue [hunk 1]
```diff
@@ -1,13 +1,7 @@
 <template>
 <form ref="fixedDeltaInput"
       class="c-conductor__inputs"
```
Comment (shefalijoshi) [inline src/plugins/timeConductor/ConductorInputsFixed.vue:3]: Nice!

**Review by shefalijoshi — CHANGES_REQUESTED** [review]
Looks good, small changes.

File: src/api/telemetry/TelemetryCollection.js [hunk 2]
```diff
@@ -103,41 +102,35 @@ export class TelemetryCollection extends EventEmitter {
...
-        this._requestHistoricalTelemetry();
-    }
-
     /**
      * If a historical provider exists, then historical requests will be made
      * @private
      */
     async _requestHistoricalTelemetry() {
-        if (!this.historicalProvider) {
+        let options = this.options ? { ...this.options } : undefined;
```
[the remark this answers is not in the document]
Comment (jvigliotta) [reply src/api/telemetry/TelemetryCollection.js:110]: good call!

File: src/api/telemetry/TelemetryCollection.js [hunk 3]
```diff
@@ -103,41 +102,35 @@ export class TelemetryCollection extends EventEmitter {
...
+        let historicalProvider;
+
+        this.openmct.telemetry.standardizeRequestOptions(options);
```
[the remark this answers is not in the document]
Comment (jvigliotta) [reply src/api/telemetry/TelemetryCollection.js:113]: I do, I'm not using `openmct.telemetry.request` I'm going straight to the provider's request.

**Review by shefalijoshi — APPROVED** [review]
LGTM
### Reviewer Checklist

* [x] Changes appear to address issue?
* [ ] Appropriate unit tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Commit messages meet standards?
* [ ] Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled `bug`? (only applicable if this PR is for a bug fix)
* [ ] List of Acceptance Tests Performed.

**Pull request merged**